tests/flow/test_node_by_id.py

```diff
@@ -226,3 +226,15 @@ def test_get_nodes(self):
         self.env.assertNotIn("NodeByIdSeek", redis_graph.execution_plan(query))
         resultsetB = redis_graph.query(query).result_set
         self.env.assertEqual(resultsetA, resultsetB)
+
+    # Try to fetch none existing entities by ID(s).
+    def test_for_none_existing_entity_ids(self):
+        # Try to fetch an entity with a none existing ID.
+        queries = ["""MATCH (a:person) WHERE ID(a) = 999 RETURN a""",
+                    """MATCH (a:person) WHERE ID(a) > 999 RETURN a""",
+                    """MATCH (a:person) WHERE ID(a) > 800 AND ID(a) < 900 RETURN a"""]
```

Comment:
We should add a test in which the iterator is constructed during `NodeByLabelScanConsumeFromChild`, such as:
`UNWIND [1,2] AS val CREATE (:A {v: val}) WITH val MATCH (a:A) WHERE ID(a) = 999 RETURN a, val`